### PR TITLE
Experimenting with not running Kiln on non-functional commits

### PR DESCRIPTION
**Pull request by BryceStevenWilley**
Tests shouldn't need to run when change the README or the CHANGELOG.

### CODE REVIEW

**Comment by nonprofittechy** [comment]
Does this stop kiln from re-running when you modify a .feature file? is that desired?

**Review by nonprofittechy — CHANGES_REQUESTED** [review]
Not sure this is a desirable or correct change

**Comment by BryceStevenWilley** [comment]
> Does this stop kiln from re-running when you modify a .feature file?

No, this line should keep it running on anything in the actual package: `'docassemble/AssemblyLine/**'`. The `*.py` should keep it running on `setpy.py` or `__init__.py` files, that could break things (but really shouldn't).

File: .github/workflows/run_form_tests.yml [hunk 1]
```diff
@@ -6,6 +6,11 @@ on:
       - "**"
     tags-ignore:
       - "**"
+    paths:
+      - 'requirements.txt'
+      - '**.py'
+      - '**.yml'
+      - 'docassemble/AssemblyLine/**'
```
Comment (nonprofittechy) [inline .github/workflows/run_form_tests.yml:13]:
```suggestion
      - 'docassemble/**'
```

**Pull request merged**